spec/broadside/deploy/ecs_deploy_spec.rb

```diff
@@ -68,11 +68,59 @@
       expect { deploy.bootstrap }.to raise_error(/Service doesn't exist and no :service_config/)
     end
 
-    it 'succeeds' do
-      deploy.deploy_config.service_config = service_config
-      deploy.deploy_config.task_definition_config = task_definition_config
+    context 'with an existing task definition and service' do
+      let(:arn) { 'arn:aws:ecs:us-east-1:1234' }
```

Comment:
Yeah... I thought about that but it would have required a pretty serious refactoring that I didn't think I really had time to tackle.